packages/web3-constants/evm/explorer.json

```diff
@@ -13,7 +13,8 @@
         "Arbitrum_Rinkeby": "",
         "xDai": "",
         "Celo": "",
-        "Fantom": "https://api.ftmscan.com/api"
+        "Fantom": "https://api.ftmscan.com/api",
+        "Avalanche": "https://api.avax.network/"
```

Comment:
It's not working. Let's leave it to empty `''`.

https://api.avax.network/?module=account&action=txlistinternal&address=0x66b57885E8E9D84742faBda0cE6E3496055b012d&startblock=0&endblock=999999999999&page=1&offset=10&sort=asc&apiKey=26GHYKIAZREN4HWJ5NCICTGKC183IGXY9A